habitat_baselines/common/rollout_storage.py

```diff
@@ -172,11 +191,12 @@ def compute_returns(self, next_value, use_gae, gamma, tau):
                     self.buffers["rewards"][step]
                     + gamma
                     * self.buffers["value_preds"][step + 1]
-                    * self.buffers["masks"][step + 1]
+                    * self.buffers["not_done_mask_0"][step + 1]
+                    * self.buffers["not_done_mask_1"][step + 1]
```

Comment:
This is the same as logical_and. Only use next step return if both not_done_mask_0 and not_done_mask_1 are true (no reset)